.github/actions/run-backend-tests/action.yml

```diff
@@ -83,7 +83,7 @@ runs:
         - uses: syphar/restore-virtualenv@v1
           id: cache-backend-tests
           with:
-              custom_cache_key_element: v1
+              custom_cache_key_element: v2
```

Comment:
Not sure if this is necessary, but CI was insisting on using an old version of structlog and hogql_parser.